src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -67,6 +67,17 @@ const WSTRING log_path = "DD_TRACE_LOG_PATH"_W;
 // https://github.com/dotnet/coreclr/issues/12468
 const WSTRING clr_disable_optimizations = "DD_CLR_DISABLE_OPTIMIZATIONS"_W;
 
+// Sets whether to intercept method calls when the caller metohd is inside a
```

Comment:
typo
```suggestion
// Sets whether to intercept method calls when the caller method is inside a
```

Reply:
Resolved in 1a79b7b5239ab20af134e26484cca6db56e29cb5